File: aspnet-core/src/Selling_Website.Application.Contracts/Customers/Dto/CustomerDto.cs
using System;
using Volo.Abp.Application.Dtos;

namespace Selling_Website.Customers.Dto
{
    public class CustomerDto : FullAuditedEntityDto<Guid>
    {
        public string Name { get; set; }
        public DateTime? DOB { get; set; }
        public string Address { get; set; }
        public string Email { get; set; }
        public string PhoneNumber { get; set; }
        public string Size { get; set; }
        public string Message { get; set; }
        public string Subject { get; set; }
    }
}

File: angular/src/app/shared/service/customer/models/customer-dto.ts
import { EntityDto } from '@abp/ng.core';

export class CustomerDto extends EntityDto<string>{
  name: string;
  dob: Date;
  address: string;
  email: string;
  phonenumber: string;
  message: string;
  subject: string;

  constructor(initial: Partial<CustomerDto> = {}) {
    super(initial);
  }
}